specs/yara/yara.table

```diff
@@ -1,5 +1,5 @@
 table_name("yara")
-description("Track YARA matches for files or PIDs.")
+description("Fetch one-off YARA matches for files or PIDs at the specified path.")
```

Comment:
Well, this has caused me to look at how things work. And while I agree your language is an improvement, I think we can be better.
1. I don't think this uses pids at all. 
2. `track` isn't right, but I don't know if `fetch` is right either. It triggers an on-demand yara scan. (Like if you queried for a hash, it's not fetching the hash, per se, it's generating it)
3. It's not listed here, but at least one of `sig_groups`, `sigfile`, or `sigrule` must be specified. I don't know if you'd like to add that.

Reply:
Updated this! WDYT?

Reply:
Updated the language! WDYT?